build.gradle

```diff
@@ -228,22 +230,30 @@ task syncFrontendPublic(type: Sync) {
 
 task macHotReloadFrontend(type: Exec) {
     dependsOn(installFrontend)
-    onlyIf {Os.isFamily(Os.FAMILY_MAC)}
+    onlyIf {os.isMacOsX()}
     workingDir 'frontend/'
     commandLine 'osascript', '-e', '"tell application "Terminal" to do script "npm run serveOpen""'
 }
 
 task windowsHotReloadFrontend(type: Exec) {
     dependsOn(installFrontend)
-    onlyIf {Os.isFamily(Os.FAMILY_WINDOWS)}
+    onlyIf {os.isWindows()}
     workingDir 'frontend/'
     commandLine 'cmd','/c', 'START', '"hotreload RepoSense frontend"', 'npm', 'run', 'serveOpen'
 }
 
+task linuxHotReloadFrontend(type: Exec) {
+    dependsOn(installFrontend)
+    onlyIf {os.isLinux()}
+    workingDir 'frontend/'
+    commandLine 'npm', 'run', 'serveOpen'
```

Comment:
This works for me, but I'm not sure if there's a canonically correct way to write this.

Reply:
There is no canonically correct way, there is only consistency, and I find it consistent other than the fact that this one doesn't spawn the `serveOpen` in a new window which may or may not be needed.